docs/code-quality/c6504.md

```diff
@@ -47,10 +53,18 @@ public:
     //  members
 };
 
-void f(_Pre_ _Null_  Point* pt)
+// Changed to pointer type because it may be null
+void f(_Pre_ _Maybenull_  Point* pt)
+{
+    // code ...
+}
+
+// Removed annotation because it did not apply to reference types.
+void g(Point& pt)
 {
     // code ...
 }
 ```
 
- The defective code shown earlier also generates warning [C6516](../code-quality/c6516.md) because property conflicts resulted in an invalid annotation.
+
+[Annotation Properties]:using-sal-annotations-to-reduce-c-cpp-code-defects.md
```

Comment:
does this work or do you need ()

Reply:
This is the markdown syntax if you are using the link as a reference in other places in the document.